asv.conf.json

```diff
@@ -0,0 +1,160 @@
+{
+    // The version of the config file format.  Do not change, unless
+    // you know what you are doing.
+    "version": 1,
+
+    // The name of the project being benchmarked
+    "project": "Cirq",
+
+    // The project's homepage
+    "project_url": "https://quantumai.google/cirq",
+
+    // The URL or local path of the source code repository for the
+    // project being benchmarked
+    "repo": ".",
+
+    // The Python project's subdirectory in your repo.  If missing or
+    // the empty string, the project is assumed to be located at the root
+    // of the repository.
+    // "repo_subdir": "",
+
+    // Customizable commands for building, installing, and
+    // uninstalling the project. See asv.conf.json documentation.
```

Comment:
Why does it say "See asv.conf.json documentation."  This file is asv.conf.json.

Reply:
This file template (and the comments) were auto-generated by asv -- but I think it's refering to the documentation online i.e. https://asv.readthedocs.io/en/stable/asv.conf.json.html#build-command-install-command-uninstall-command

Reply:
modified the comment to include the link.